lti_consumer/plugin/compat.py

```diff
@@ -291,3 +291,16 @@ def nrps_pii_disallowed():
     """
     return (hasattr(settings, 'LTI_NRPS_DISALLOW_PII') and
             settings.LTI_NRPS_DISALLOW_PII is True)
+
+
+def get_enable_processor():
+    """
+    Get the enable processor setting for LTI Consumer XBlock from Site Configurations.
+    """
+    # pylint: disable=import-error,import-outside-toplevel
+    from openedx.core.djangoapps.site_configuration import helpers
+    enable_processor = helpers.get_current_site_configuration().get_value(
+        'ENABLE_LTI_CONSUMER_XBLOCK_PROCESSOR',
+        default=False,
+    )
+    return bool(enable_processor)
```

Comment:
```suggestion
def are_processors_enabled():
    """
    Get 'LTI_CONSUMER_XBLOCK_ENABLE_PROCESSORS' setting from site configurations.
    to determine if LTI Consumer XBlock processors should be enabled site-wide.
    
    Returns:
        True if setting is enabled.
        False if setting is disabled.
        
    """
    # pylint: disable=import-error,import-outside-toplevel
    from openedx.core.djangoapps.site_configuration import helpers
    
    return helpers.get_current_site_configuration().get_value(
        'LTI_CONSUMER_XBLOCK_ENABLE_PROCESSORS',
        default=False,
    )

```
@JuanDavidBuitrago What do you think about rephrasing the function name?, I also don't think is necessary to cast the setting, doesn't the helper cast the value already?, if not then I think it's OK to cast it.

Reply:
Yes, I think the name that you propose works better.